libflux/src/flux/semantic/import.rs

```diff
@@ -0,0 +1,5 @@
+pub trait Importer<T> {
```

Comment:
Why is Importer parametric on the type it imports? It looks like all uses are for the `PolyType` type. Is that correct? In that case does it make sense to change this to be more specific?